.github/workflows/publish_pytorch_dev_docker.yml

```diff
@@ -105,7 +105,7 @@ jobs:
       - name: Trigger verification workflow
         uses: benc-uk/workflow-dispatch@e2e5e9a103e331dad343f381a29e654aea3cf8fc #1.2.4
         with:
-          workflow: verify_docker_image.yml
+          workflow: test_pytorch_docker.yml
           inputs: |
```

Comment:
Unrelated to this renaming, but is the `dispatch_id` input actually used? It doesn't appear to be.

Reply:
I am aware of it. I will raise another PR to improve  `test_pytorch_docker.yml`.  `dispatch_id`  is important when you build multiple docker images to match with the correct  `test_pytorch_docker.yml`

Reply:
I'm not seeing any uses right now. If code isn't used, please delete it.

https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it

Reply:
Sounds good will added later.